nan.h

```diff
@@ -1745,10 +1745,47 @@ typedef AsyncProgressWorkerBase<char> AsyncProgressWorker;
 
 template<class T>
 /* abstract */
-class AsyncProgressQueueWorker : public AsyncBareProgressWorker<T> {
+class AsyncBareProgressQueueWorker : public AsyncBareProgressWorkerBase {
+ public:
+  explicit AsyncBareProgressQueueWorker(Callback *callback_)
+      : AsyncBareProgressWorkerBase(callback_) {
+  }
+
+  virtual ~AsyncBareProgressQueueWorker() {
+  }
+
+  class ExecutionProgress {
+    friend class AsyncBareProgressQueueWorker;
+   public:
+    void Send(const T* data, size_t count) const {
+        that_->SendProgress_(data, count);
```

Comment:
Indent error here and below.

Reply:
fixed in 76bee55889ed4ec3df42ba49c3e16acbbc7078d1  ...